app/src/assets/app.extensions.json

```diff
@@ -331,7 +331,7 @@
         "order": 100,
         "data": {
           "iconButton": true,
-          "focusedElementOnCloseSelector": "#share-action-button"
+          "autoFocusedTriggerAfterModalClose": true
```

Comment:
is it possible to make it `true` by default and avoid setting it for each entry here?

Reply:
Well it was my first go before I change selector to boolean but when I remove that flag and assume that default is true, it breaks some options without modals. So I decided to leave that boolean to avoid more changes. 

But sure I will check it again to see how many extra changes is required to remove that flag. If there won't be too many then sure I will remove that flag. I will let you know when I check.

Reply:
@DenysVuika I removed these flags successfully so now there is no flags in json file.